lib/cldr/backend.ex

```diff
@@ -599,14 +829,36 @@ defmodule Cldr.Territory.Backend do
                   {:ok, (unquote(Macro.escape(territories))[code][style])}
               end
           end
+
+          def translate_subdivision(localized_string, locale_from, unquote(locale_name)) do
+            locale_from
+            |> Cldr.validate_locale(unquote(backend))
+            |> case do
+                {:error, reason} -> {:error, reason}
+
+                {:ok, %LanguageTag{cldr_locale_name: locale}} ->
+                  {code, _} = locale
+                              |> Cldr.Config.get_locale(unquote(backend))
```

Comment:
This is a very expensive call and should really only be called at compile time - not at runtime.

Reply:
Thanks, I’ll refactor this before the next release.

Reply:
I was able to make a very simple change to speed up things.

So instead of calling `Cldr.Config.get_locale(locale, unquote(backend))` on runtime, then we call: `known_subdivisions(locale)` or `known_territories(locale)` that are created on compile time.


## Before
```
Operating System: macOS
CPU Information: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
Number of Available Cores: 12
Available memory: 16 GB
Elixir 1.11.0
Erlang 23.1.1

Benchmark suite executing with the following configuration:
warmup: 2 s
time: 10 s
memory time: 2 s
parallel: 1
inputs: none specified
Estimated total run time: 14 s

Benchmarking translate_subdivision/4...

Name                              ips        average  deviation         median         99th %
translate_subdivision/4          0.23         4.28 s     ±1.47%         4.24 s         4.35 s

Memory usage statistics:

Name                       Memory usage
translate_subdivision/4         1.28 GB
```

## After
```
Operating System: macOS
CPU Information: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
Number of Available Cores: 12
Available memory: 16 GB
Elixir 1.11.0
Erlang 23.1.1

Benchmark suite executing with the following configuration:
warmup: 2 s
time: 10 s
memory time: 2 s
parallel: 1
inputs: none specified
Estimated total run time: 14 s

Benchmarking translate_subdivision/4...

Name                              ips        average  deviation         median         99th %
translate_subdivision/4         24.56       40.72 ms    ±12.85%       39.04 ms       58.55 ms

Memory usage statistics:

Name                       Memory usage
translate_subdivision/4        26.91 MB
```

Reply:
That still seems way to slow for a lookup. I'm just finishing up some other `ex_cldr` polish and will take a look at this in the next day or so.